### PR TITLE
Rebuild 2024/07/09

### PR DESCRIPTION
**Pull request by WebMaestroFr**
Update from fork upstream, and rebuild of the `.db` files.

### CODE REVIEW

**Comment by WebMaestroFr** [comment]
image
While I don't have access to WEL, this update includes the missing data.

File: modules/wel/module.json [hunk 1]
```diff
@@ -37,7 +37,16 @@
...
+	"packs": [
+		{
+			"name": "scenes",
+			"label": "WEL Scenes",
+			"path": "packs/scenes.db",
+			"entity": "Scene",
+			"type": "Scene",
+			"system": "dnd5e"
+		}
+	],
```
Comment (WebMaestroFr) [inline modules/wel/module.json:49]: Added scenes for WEL.

**Comment by WebMaestroFr** [comment]
Both [#29 DDB Converter update](https://github.com/ForgeVTT/dndbconverter/pull/29/files) and [#10 DDB Metadata rebuild](https://github.com/ForgeVTT/ddb-meta-data/pull/10) were merged onto the Forge with [#2061](https://github.com/ForgeVTT/theforge/pull/2061).
Can someone please review and merge these submodule updates so the main branches match and don't go stale?

**Review by kakaroto — APPROVED** [review]
I don't think there's really much to review, is there? You just merged upstream and rebuilt the db files.
I did check the two new modules and them module.json but only gave a cursory glance.
LGTM unless you say there was something specific I should look at.

**Pull request merged**